graylog2-web-interface/src/components/cluster/index.js

```diff
@@ -0,0 +1 @@
+export GraylogClusterOverview from './GraylogClusterOverview';
```

Comment:
I remember IntelliJ didn't like some of them (was highlighting them in red), but don't remember about that. Anyway, I will update this one on the PR and change others I can find in master. In that way we don't copy them around like in this case 👍